dev/com.ibm.ws.concurrent_fat_reactive/test-applications/reactiveapp/src/reactiveapp/web/ThreadSubscriber.java

```diff
@@ -23,7 +24,13 @@
 public class ThreadSubscriber implements Flow.Subscriber<ContextCDL> {
 
     private Flow.Subscription subscription = null;
-    private Throwable closedException = null;
+    public Throwable completeException = null;
+    public Throwable errorException = null;
+
+    @SuppressWarnings("serial")
+    public class PassException extends Throwable {
+
+    }
```

Comment:
This works, but is a little bit confusing with creating an artificial exception to indicate a passing result. It might be more clear to have `Object onCompleteResult` and `Object onErrorResult`, which could be either a successful result such as the successfully looked up instance or, in the case of a failure, the failure exception.

Reply:
I wasn't a huge fan of my solution, and this is much better. I'll get this change made